src/spec.md

```diff
@@ -79,6 +79,49 @@ Each `update` which is passed to the callback comes with the following propertie
 - `update.summary`: optional, short text, shown beside icon (see [`sendUpdate()`](#sendupdate))
 
 
+### sendToChat()
+
+```js
+promise = window.webxdc.sendToChat(message);
+```
+
+`sendToChat()` (requires Delta Chat 1.38 or compatible) allows an webxdc app to prepare a message
```

Comment:
maybe we should think about an upgrading path and mention somewhere that you can just check if the method / property exists as an compatibility check:
```
if(window.webxdc.sendToChat){
/* api exists*/
} else {
/* api does not exist, ask user to update their client or provide alternatives or both */
}
```

Reply:
+1, i also thought about that, however, that also seems to be widely known in js land - and the api is not in wide use anyway currently, so i forget about that. but feel free to do a pr